client/src/pages/EpisodeDetailsPage.js

```diff
@@ -155,16 +174,18 @@ function EpisodeDetailsPage() {
               " | " +
               episodeDetails.duration_min +
               " Min. | " +
-              episodeDetails.likes +
+              likeCount +
               " Likes"}
           </Stats>
           <ButtonWrapper>
             <AddToSpotifyButton
               as="a"
               href={episodeDetails.external_urls.spotify}
+              target="_blank"
+              rel="noopener noreferrer"
             />
             <button onClick={handleClickLike}>
-              {episodeDetails.liked ? <LikedSmall /> : <NotLikedSmall />}
+              {isLiked ? <LikedSmall /> : <NotLikedSmall />}
             </button>
           </ButtonWrapper>
         </>
```

Comment:
Cannot directly comment on the prop types: Could some of them be probably required?

Reply:
Good hint, propTypes were actually at the wrong place here and needed to be moved :) and some of them can also be required